src/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Also add this to the readme instructions for those who will reproduce the build

Reply:
That sounds like the voice of cmake experience;)